+++ sandbox/freeradius-2.1.10+dfsg/debian/freeradius.postinst	2012-02-06 13:03:21.000000000 +0100
@@ -114,7 +114,7 @@
         # install or an upgrade from before there were links; users may
         # want to remove them...
         if [ -z "$2" ] || dpkg --compare-versions "$2" lt 2.0.4+dfsg-4; then
-          for site in default inner-tunnel; do
+          for site in linotp; do
             if [ ! -e /etc/freeradius/sites-enabled/$site ]; then
               ln -s ../sites-available/$site /etc/freeradius/sites-enabled/$site
             fi
